.changeset/lazy-years-share.md

```diff
@@ -0,0 +1,5 @@
+---
+"fuels": patch
```

Comment:
```suggestion
```

Reply:
This was added because of the changes [here](https://github.com/FuelLabs/fuels-ts/pull/3297/files#diff-4704641bb82aae769ecd36af59ab32785ec1fce719c6d3f6bfcd2090843fd54eR18-R33) CI would [fail](https://github.com/FuelLabs/fuels-ts/actions/runs/11553865439/job/32155914684?pr=3297) otherwise

Reply:
I see - I thought `.test.ts` files would be skipped. 

I'll resolve, nice one.